introduction.md

```diff
@@ -1 +1,5 @@
 # Hi this is John, and my favourite codeschool path is Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?